### ISSUE TITLE
Very Simple Composer as main composer

### ISSUE THREAD

**Issue by lampholder** [vector-im/riot-web#7697]
Implement an alternative Very Simple Composer that doesn't have all of the rich text goodness but is optmised for accessibility and compatibilty.

This could be as simple as just a textarea (it would be nice if we still had some autocomplete functionality there).

A bit more thought required on this before pursuing.

**Comment by lampholder** [comment]
As a very first cut, I think it would be nice if we could implement a Very Simple Composer that:
- can be toggled in settings (when enabled it replaces the slate composer entirely)
- is a close as possible to 'just a textarea'
- doesn't support markdown
- doesn't support any pills
- relies on the OS emoji picker
- all pastes are rendered as the most nieve plaintext renderings of the clipboard material (i.e. if you paste an html link it renders as just the text of the link)

And as a stretch goal it could support:
- autocompleting of:
    - fragments of display names to full display names (which once completed are just plain ol' text) and
    - fragments of room names to full room names (as above)
- a markdown toggle that decides whether the entered text is submitted to matrix as plain text or via a md interpreter and submitted as HTML
- pasting images doing the 'right think' (uploading the clipboard image as a file)

**Comment by lampholder** [comment]
@ara4n (or others) - thoughts?

**Comment by jryans** [comment]
This is nice to have as part of the aggregation project if it's easy to use the message editing composer (https://github.com/vector-im/riot-web/issues/9481) as the main one as well.

**Comment by Ai-rin** [comment]
Does it really need to exclude markdown support by default?  Friendica and Diaspora, for example, have message editors with formatting support, that 100 times faster than Riot's one. 
Can't wait for something to replace or fix it, it's insanely slow currently :(

**Comment by bwindels** [comment]
Closing this in favor of https://github.com/vector-im/riot-web/issues/9866
